pkg/crc/preflight/preflight_darwin.go

```diff
@@ -107,24 +107,24 @@ var traySetupChecks = [...]Check{
 }
 
 func getAllPreflightChecks() []Check {
-	return getPreflightChecks(true, network.DefaultMode)
+	return getPreflightChecks(true, true, network.DefaultMode)
 }
 
-func getPreflightChecks(experimentalFeatures bool, mode network.Mode) []Check {
+func getPreflightChecks(experimentalFeatures bool, trayAutostart bool, mode network.Mode) []Check {
```

Comment:
Here `experimentalFeatures` is no longer used so better to `_ bool` ?

Reply:
Done